DuggaSys/tests/courseedservice-test.php

```diff
@@ -0,0 +1,373 @@
+/*
+----------------------------------------------------------
+    Courseedservice.php implement test file
+----------------------------------------------------------
+*/
+ 
+<?php
+ 
+include "../../Shared/test.php";
+ 
+$testsData = array(
+
+    /* Insert into course */
+
+    'Insert into course test' => array(
```

Comment:
the expected output is wrong.. and i have already supplied an example of how it should look..